examples/Microsoft.Spark.CSharp.Examples/README.md

```diff
@@ -29,6 +29,9 @@ an open source and cross-platform machine learning framework.
   <tr>
     <td align="middle"><a href="Sql/Batch/VectorUdfs.cs"><b>VectorUdfs.cs</a></b><br>Example using vectorized UDFs to improve query performance.<br></td>
   </tr>
+  <tr>
+    <td align="middle"><a href="Sql/Batch/VectorDataFrameUdfs.cs"><b>VectorDataFrameUdfs.cs</a></b><br>Example using vectorized UDFs and convenience APIs from Microsft.Data.Analysis to improve query performance.<br></td>
```

Comment:
this should go into `<td>` since we have two cols (or use `colspan="2"`)? You can see the output by looking at the rich diff.